README.md

```diff
@@ -76,6 +76,36 @@ public class JfrTest {
 }
 ```
 
+Or with Spock Framework like this:
+```groovy
+import dev.morling.jfrunit.*
+import spock.lang.Specification
+
+import java.time.Duration
+
+import static dev.morling.jfrunit.JfrEventsAssert.*
+import static dev.morling.jfrunit.ExpectedEvent.*
+
+@JfrEventTest
+class JfrSpec extends Specification {
```

Comment:
Adding an actual test would be a good idea, and I'd also suggest to test a data-driven feature in combination with this extension.

Reply:
I'll try to add some test later tonight/tomorrow.

Reply:
Leonard, @gunnarmorling I've added more tests.